launcher/minecraft/VersionFilterData.cpp

```diff
@@ -70,4 +70,7 @@ VersionFilterData::VersionFilterData()
     java16BeginsDate    = timeFromS3Time("2021-05-12T11:19:15+00:00");
     java17BeginsDate    = timeFromS3Time("2021-11-16T17:04:48+00:00");
     quickPlayBeginsDate = timeFromS3Time("2023-04-05T12:05:17+00:00");
+    liteLoaderEndsDate  = timeFromS3Time("2017-09-18T08:39:46+00:00");
+    fabricBeginsDate    = timeFromS3Time("2019-04-23T14:52:44+00:00");
+    neoForgeBeginsDate  = timeFromS3Time("2023-09-20T09:02:57+00:00");
```

Comment:
earliest neoforge is 1.20.1 https://github.com/phit/multimc-meta-upstream/blob/develop/neoforge/version_manifests/1.20.1-47.1.5.json#L4

Reply:
I already fixed that https://github.com/MultiMC/Launcher/pull/5323/commits/4fe664204e87d562172153d874985d1a3cda2c79